src/test/java/eu/dissco/core/digitalspecimenprocessor/service/KafkaConsumerServiceTest.java

```diff
@@ -83,73 +61,72 @@ private String givenMessage() {
             "type": "GeologyRockSpecimen",
             "physicalSpecimenId": "https://geocollections.info/specimen/23602",
             "physicalSpecimenIdType": "cetaf",
-            "specimenName": "Biota",
+            "specimenName": "Biota,
```

Comment:
Yes this is an invalid message because some idiot forgot a quotation mark there! We can't accept those messages! (That why the test is called givenInvalidMessage() :wink: )